### ISSUE TITLE
What is the correct process for connecting two databases

### ISSUE THREAD

**Issue by HeWhoWatches** [keepassxreboot/keepassxc-browser#621]
## Expected Behavior

Under Connected Databases, I'd expect the Connect button to attempt a connection to the currently selected database in KeePassXC, in order to allow setting up more than one database.

## Current Behavior

I can only ever seen to make a connection to a single database. Making a connection is fine (to either database), but once one of the two databases has been connected, the Connect button doesn't do anything. I'm not sure what the correct process is for connecting the second database (have looked on GitHub including through existing issues but I'm none the wiser, apart from anecdotal evidence that other people are able to make this work).

## Possible Solution

I'm probably just doing something wrong but some advice would be much appreciated! :)

## Steps to Reproduce (for bugs)

1. Launch Firefox (tested with clean profile) and install the extension.
2. Connect the first database and refresh the extension - connected database is displayed and passwords work as expected.
3. Try to connect second database - Connect button does nothing, regardless of whether second database is selected in KeePassXC or not.
4. It is possible to connect both databases one at a time by closing Firefox and KeePassXC down after connecting the first database, then connecting to the second database. However it seems the browser will only connect to one of the two databases regardless of whether they are both open in KeePassXC (Search in all opened databases is enabled under Browser Integration > Advanced, and Automatically reconnect is set in the extension settings).

## Debug info

KeePassXC - 2.43
KeePassXC-Browser - 1.5.1
Operating system: Windows 10
Browser: Firefox 69.0 (but tested in Chrome 77.0.3865.90 as well and can't connect a second database here either)

**Comment by varjolintu** [comment]
This is the workflow when the extension is connected to KeePassXC:
1. Select the first database and open it, open the extension popup using the icon and select Connect. Give the database connection a name.
2. Select the second database, open the extension popup, and it should show the Connect button again. Click it and give the second database a name.

You don't need to restart the extension, browser or KeePassXC between the connections.

**Comment by HeWhoWatches** [comment]
Thanks varjolintu, I've given this a try and step 1 works fine but nothing happens when clicking the Connect button at step 2?

**Comment by varjolintu** [comment]
Any difference if you try the latest snapshot build? You can get it from https://snapshot.keepassxc.org/latest/

**Comment by HeWhoWatches** [comment]
Hi varjolintu, I was eventually able to get this working with both the current and snapshot versions... One of the databases just wouldn't work (created a fresh database for comparison and that _did_ work) so exported that as a CSV and reimported as a new database, and it seems that I can now connect to both databases in Firefox and Chrome. Just some notes...
* Once I had managed to get it working in a clean Firefox profile, I still had to remove and reinstall the extension in my normal profile to get it to work as expected.
* For completeness, I noticed when comparing against a fresh database that Security > Encryption Settings > Decryption Time: was set to "unchanged" rather than the default of 1.0s, so I did set this as well (though I doubt it made any difference).
* I couldn't see any other settings in the database I suspect was corrupt in some way that looked incorrect. Possibly something to do with the way the database was originally created (possibly in KeePass or KeePassX, it's been around for a while).

Fingers crossed it was just some sort of incompatibility in the database (and possibly Firefox extension) due to the age of the setup?

Thanks very much for your help, much appreciated - I'll close the issue and monitor on my side.
